NuRadioReco/modules/io/coreas/readCoREASShower.py

```diff
@@ -177,17 +177,4 @@ def end(self):
             timedelta(seconds=self.__t_event_structure)))
         logger.info("per event {}".format(
             timedelta(seconds=self.__t_per_event)))
-        return dt
-
-
-def antenna_id(antenna_name, default_id):
-    """
-    This function parses the antenna names given in a CoREAS simulation and tries to find an ID
-    It can be extended to other name patterns
-    """
-
-    if re.match("AERA_", antenna_name):
-        new_id = int(antenna_name.strip("AERA_"))
-        return new_id
-    else:
-        return default_id
```

Comment:
This is actually handy/useful when simulating events for a real array.

Reply:
Can you add that again? :)

Reply:
I think I will add it later if I need it again. No need for this PR